README.md

```diff
@@ -1,3 +1,106 @@
-# Rainforest QA Github Action
+# Rainforest QA GitHub Action ![](https://img.shields.io/github/v/release/rainforestapp/rainforest-github-action.svg)
 
-This is currently a WIP.
\ No newline at end of file
+**Marketplace homepage:** [`rainforestapp/rainforest-github-action`](https://github.com/marketplace/actions/rainforest-qa)
+
+> This is the Rainforest QA [GitHub Action](https://docs.github.com/en/actions), it allows you to easily kick off a Rainforest run from your GitHub workflows, to make sure that every release passes your Rainforest integration tests.
+
+## Sections
+* [Prerequisites](#prerequisites)
+* [Base Usage](#base-usage)
+* [Optional Parameters](#optional-parameters)
+* [Release Process](#github-action-release-process)
+
+## Prerequisites
+
+### A Rainforest QA account
+
+If you don't already have one, you can sign up for a free account [here](https://app.rainforestqa.com/auth/signup).
+
+### A Rainforest QA API token
+
+You can find yours on the [Integrations](https://app.rainforestqa.com/settings/integrations) setting page.
+Do not expose this token in your `.github/workflows/<workflow>.yml` file. Instead, [use a GitHub encrypted secret](https://docs.github.com/en/actions/security-guides/encrypted-secrets). You may name this secret as you wish (as long as it's a name allowed by GitHub), e.g. `RAINFOREST_API_TOKEN`.
+
+### A run group with at least one test
+
+Run groups are a way to group tests that should be run together (for example, a smoke suite you might want to run on every deploy). For more information on run groups, see [this help article](https://help.rainforestqa.com/docs/organizing-tests-by-run-group).
+
+Once you have a run group which contains at least one test (included either directly in the run group or via a Feature added to the run group), you can run it in GitHub workflows using this Action. You will need its ID (visible at the end of the run group URL: `https://app.rainforestqa.com/run_groups/<ID>`).
+
+## Base usage
+
+```yaml
+# .github/workflows/<workflow>.yml
```

Comment:
Remove this line if you decide to add the more explicit instructions I suggested above?

```suggestion
```